### ISSUE TITLE
New strategy available: passport-negotiate implementing Negotiate (kerberos/SPNEGO)

### ISSUE THREAD

**Issue by dmansfield** [jaredhanson/passport#408]
The passport-negotiate module is now available which implements Negotiate (kerberos/SPNEGO) authentication.  Can it be added to the "available strategies" page on the main passport website?

npm: https://www.npmjs.com/package/passport-negotiate
github: https://github.com/dmansfield/passport-negotiate

**Comment by jaredhanson** [comment]
Instead of depending on passport@2.0.0, can you depend on [passport-strategy](https://github.com/jaredhanson/passport-strategy)?  That's important for compatibility with newer versions of Passport, including 0.3.x and on.

**Comment by dmansfield** [comment]
Is there a specific version I should target?

**Comment by jaredhanson** [comment]
Just inherit from passport-strategy:

https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L4
https://github.com/jaredhanson/passport-oauth2/blob/master/package.json#L35

**Comment by jaredhanson** [comment]
Passport strategies should not depend on Passport core.  They should only depend on `passport-strategy`.

**Comment by dmansfield** [comment]
v0.1.3 has been published with this change.  I have re-tested with the included "login" example app.

**Comment by dmansfield** [comment]
Hey Jared.  Dependencies have been fixed as per your suggestions. Any other comments regarding the module?  Any chance to add it to the "strategies" section on the site?

**Comment by zckrs** [comment]
Hi
I try to use `passport-negotiate`. But I would know if this module is maintained.

**Comment by dmansfield** [comment]
Yes, it is maintained. Unfortunately the underlying module for kerberos doesn't support windows at this time - that is the only known issue.  If you have any issues please feel free to add them to the issue tracker for the repository: https://github.com/dmansfield/passport-negotiate
